test/benchmarks/benchmark_slidehash.cc

```
@@ -92,6 +92,9 @@ BENCHMARK_SLIDEHASH(armv6, slide_hash_armv6, test_cpu_features.arm.has_simd);
 #ifdef ARM_NEON
 BENCHMARK_SLIDEHASH(neon, slide_hash_neon, test_cpu_features.arm.has_neon);
 #endif
+#ifdef MIPS_MSA
+BENCHMARK_SLIDEHASH(msa, slide_hash_msa, test_cpu_features.mips_.has_msa);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If BENCHMARK_DEFINE_F is a macro then please configure it.

(unknownMacro)